receiver/k8sclusterreceiver/internal/demonset/metadata.yaml

```diff
@@ -0,0 +1,52 @@
+type: k8s/job
```

Comment:
```suggestion
type: k8s/daemonset
```

Reply:
@crobert-1 or @povilasv, can you please apply the suggestion?